<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBDScreenWindowClass</class>
 <widget class="QWidget" name="GBDScreenWindowClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>180</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Screen</string>
  </property>
  <widget class="QTabWidget" name="m_tabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="m_tabScreen">
    <attribute name="title">
     <string>Screen</string>
    </attribute>
    <widget class="QCheckBox" name="m_showTileGrid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Grid</string>
     </property>
    </widget>
    <widget class="QLabel" name="m_tileCoord">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="m_frameScreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>320</width>
       <height>288</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="gbd::ScreenWidget" name="m_screen" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>288</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="m_tabTilePatterns">
    <attribute name="title">
     <string>Tile Patterns</string>
    </attribute>
    <widget class="QFrame" name="m_patternFrameZoom">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="gbd::TilePatternZoomWidget" name="m_patternZoom" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="m_pattern0Coord">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>300</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="m_pattern0Frame">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>281</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="gbd::TilePatternWidget" name="m_pattern0" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="m_pattern1Frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="gbd::TilePatternWidget" name="m_pattern1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="m_pattern1Coord">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="m_tabSprites">
    <attribute name="title">
     <string>Sprites</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gbd::ScreenWidget</class>
   <extends>QWidget</extends>
   <header>gbd_screenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gbd::TilePatternWidget</class>
   <extends>QWidget</extends>
   <header>gbd_tilepatternwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gbd::TilePatternZoomWidget</class>
   <extends>QWidget</extends>
   <header>gbd_tilepatternzoomwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
